lib/gitlib.sh

```diff
@@ -20,8 +20,13 @@
 ###############################################################################
 # CONSTANTS
 ###############################################################################
+
 GITHUB_TOKEN=${FLAGS_github_token:-$GITHUB_TOKEN}
-[[ -n $GITHUB_TOKEN ]] && GITHUB_TOKEN_FLAG=("-u" "$GITHUB_TOKEN:x-oauth-basic")
+
+# This is a safety measure to trim extraneous leading/trailing whitespace from
+# the OAuth token provided. Otherwise, authentication to GitHub will fail here.
+GITHUB_TOKEN="$( echo "$GITHUB_TOKEN" | tr -d '[:space:]' )"
```

Comment:
_technically_ this will trim whitespace anywhere in the string, but there shouldn't be any in the middle...